wig/wig.go

```diff
@@ -139,3 +139,23 @@ func WriteToFileHandle(file io.Writer, rec Wig) {
 
 	}
 }
+
+// ChromToSlice returns the values from a wig entry matching a user-specified chromosome name
+func ChromToSlice(w []Wig, chrom string) []float64 {
+	var output []float64
+	for _, v := range w {
+		if v.Chrom == chrom {
+			if v.Step == 1 {
+				if v.Start == 0 {
```

Comment:
Doesn't the wig start at 1 in the test example? I don't see the wig.Read() turning the wig.Start into a 0 if it is a 1, but I could be missing it?>

Reply:
Test fails on the branch with error code "start coordinate invalid."